docs/overview.rst

```diff
@@ -0,0 +1,11 @@
+Overview
+========
+
+This is a library of ``pysat`` instrument modules and methods designed to support
+NASA instruments and missions archived at the Community Data Analysis Web
```

Comment:
Needs to be correct.

Reply:
caught in the index, but not the overview. 🤦